### ISSUE TITLE
New welcome page

### ISSUE THREAD

**Issue by SamuelWei** [THM-Health/PILOS#1216]
The current landing page is based on the design of Greenlight v2 and describes a mixture of the functions of the PILOS room management system and the BigBlueButton video conferencing system.

A good landing page would be inviting and clearly communicate what users can do with this product. Since the separation of PILOS and BBB is probably irrelevant for the user, consideration must be given to how this is presented, whether it is separated or marketed as a complete product.

This issue is intended to collect ideas and suggestions as well as design proposals from the community for a redesign.

**Comment by SamuelWei** [comment]
Latest design of the new landing page
![image](https://github.com/user-attachments/assets/64692000-2470-46a2-a959-03a8055cabf9)

We are looking for feedback on the standard texts and would be pleased if you give us feedback or suggest other texts before we start localization.

**Comment by danimo** [comment]
I love the design. Some Feedback from our team

* This looks like its a generic service usable by everyone. Even with the logo replaced, it's still very generic. It would be nice to have the option to add some sort of reference to the institution, like a name and link to the help page.
 
(I know its customizable by replacing the components, but it would be nice anyway)

**Comment by SamuelWei** [comment]
@danimo Thanks for your feedback, could you please show your suggested changes in a mockup? (simply draw on top of the screenshot)

**Comment by danimo** [comment]
Summary from a private discussion: What I'd ideally like to do would most certainly require a CMS just for the landing page. Let's stay with overriding components and translations for now.
